src/module-2/bootstrap-design/index.html

```diff
@@ -0,0 +1,431 @@
+            Online Store
+          </a>
+        </div>
+      </nav>
+
+
+      <nav class="container-fluid" style="--bs-breadcrumb-divider: url(&#34;data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='8'%3E%3Cpath d='M2.5 0L1 1.5 3.5 4 1 6.5 2.5 8l4-4-4-4z' fill='currentColor'/%3E%3C/svg%3E&#34;);" aria-label="breadcrumb">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item">Home</li>
+          <li class="breadcrumb-item active" aria-current="page">Ecommerce</li>
+          <li class="breadcrumb-item active" aria-current="page">Electronics</li>
+        </ol>
+      </nav>
+
+      <section class="main">
+          <div class="container-fluid row m-0">
+              <div class="col-3 d-flex justify-content-between flex-column d-md-block d-none d-sm-none">
+                  <div class="col-12">
+                      <div class="filters__header d-flex justify-content-between">
+                        Filters
+                        <button type="button" class="btn btn-light"><<</button>
+                      </div>
+                  <div class="d-flex flex-column" role="group" aria-label="Basic radio toggle button group">
+                      Multi Range
+                      <div class="price__item">
+                        <input type="radio" class="form-check-input" name="btnradio" id="btnradio1" autocomplete="off" checked>
+                        <label class="form-check-label" for="btnradio1">$10</label>
+                      </div>
+                      <div class="price__item">
+                        <input type="radio" class="form-check-input" name="btnradio" id="btnradio2" autocomplete="off">
+                        <label class="form-check-label" for="btnradio2">$10-$100</label>
+                      </div>
+                      <div class="price__item">
+                        <input type="radio" class="form-check-input" name="btnradio" id="btnradio3" autocomplete="off">
+                        <label class="form-check-label" for="btnradio3">$100-$500</label>
+                      </div>
+                      <div class="price__item">
+                        <input type="radio" class="form-check-input" name="btnradio" id="btnradio4" autocomplete="off">
+                        <label class="form-check-label" for="btnradio4">$500</label>
+                      </div>
+                      <div class="price__item">
+                        <input type="radio" class="form-check-input" name="btnradio" id="btnradio5" autocomplete="off">
+                        <label class="form-check-label" for="btnradio5">All</label>
+                      </div>
+                    </div>
+                    <div class="btn-group d-flex flex-column mt-5" role="group" aria-label="Basic checkbox toggle button group">
+                        Category
+                        <div class="category__item d-flex justify-content-between">
```

Comment:
please remove all extra classes, currently you don't have any custom css that would use them